DocumentedExamples/SeaIce_Obs_Model_Compare.ipynb

```diff
@@ -0,0 +1,5185 @@
+{
```

Comment:
<p>or could use <code>hi</code> which is already summed over the categories </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
<p>I think because hi is not in the documentation. (I assume its ice volume divided by ice area).</p><p> </p><p>So vice=hi*aice*area_t ?</p>

Reply:
<p><code>hi</code> is ice volume per unit grid cell area (not ice area - see <a href="https://github.com/COSIMA/cice5/issues/61#issuecomment-888711578)," target="_blank">https://github.com/COSIMA/cice5/issues/61#issuecomment-888711578),</a> so ice volume in each cell is <code>hi*area_t</code>.</p><p> </p><p>So <code>hi</code> is the sum of <code>vicen</code> over all categories (you might want to check this).</p><p> </p>